### ISSUE TITLE
Data cannot be read in by collect_hub

### ISSUE THREAD

**Issue by nickreich** [reichlab/variant-nowcast-hub#184]
``` r
library(hubData)
hub_con <- connect_hub(hub_path = "~/Documents/research-versioned/variant-nowcast-hub/")
hub_con |> 
  collect_hub()
#> Error in `dplyr::collect()`:
#> ! Invalid: Failed to parse string: 'DE72' as a scalar of type
#>   date32[day]
```

<sup>Created on 2024-11-26 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**Comment by zkamvar** [comment]
You may have an outdated version of hubData (we released yesterday) or you might be on a different branch? I cannot reproduce, unfortunately. 

``` r
tmp <- tempfile()
gert::git_clone("https://github.com/reichlab/variant-nowcast-hub", tmp)
hubData::connect_hub(tmp) |> hubData::collect_hub()
#> # A tibble: 17,733,408 × 8
#>    model_id   nowcast_date target_date location clade output_type output_type_id
#>  * <chr>      <date>       <date>      <chr>    <chr> <chr>       <chr>         
#>  1 CADPH-CAT… 2024-10-30   2024-09-29  CA       24A   mean        <NA>          
#>  2 CADPH-CAT… 2024-10-30   2024-09-29  CA       24B   mean        <NA>          
#>  3 CADPH-CAT… 2024-10-30   2024-09-29  CA       24C   mean        <NA>          
#>  4 CADPH-CAT… 2024-10-30   2024-09-29  CA       24E   mean        <NA>          
#>  5 CADPH-CAT… 2024-10-30   2024-09-29  CA       24F   mean        <NA>          
#>  6 CADPH-CAT… 2024-10-30   2024-09-29  CA       24G   mean        <NA>          
#>  7 CADPH-CAT… 2024-10-30   2024-09-29  CA       other mean        <NA>          
#>  8 CADPH-CAT… 2024-10-30   2024-09-29  CA       reco… mean        <NA>          
#>  9 CADPH-CAT… 2024-10-30   2024-09-30  CA       24A   mean        <NA>          
#> 10 CADPH-CAT… 2024-10-30   2024-09-30  CA       24B   mean        <NA>          
#> # ℹ 17,733,398 more rows
#> # ℹ 1 more variable: value <dbl>
```

<sup>Created on 2024-11-26 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**Comment by zkamvar** [comment]
if you use `options(error = recover)`, and run the above command, you will be put into debug mode when the error hits. You can then enter any of the listed frames and explore what the objects in the environment are. This would lead you to the offender.

**Comment by nickreich** [comment]
Yeah, i had hubData v1.2.2. my bad.

**Comment by zkamvar** [comment]
Note that this was a specific bug that was fixed in hubData v1.2.3: https://hubverse-org.github.io/hubData/news/index.html#hubdata-123
